perun/cli.py

```diff
@@ -1188,6 +1193,131 @@ def temp_sync():
+              help='Do not show the minor version headers in the output.')
+@click.option('--no-file-size', '-f', flag_value=True, default=False,
+              help='Do not show the size of each stat file.')
+@click.option('--no-total-size', '-t', flag_value=True, default=False,
+              help='Do not show the total size of all the stat files combined.')
+@click.option('--sort-by-size', '-s', flag_value=True, default=False,
+              help='Sort the files by size instead of the minor versions order.')
+@click.option('--no-color', '-c', flag_value=True, default=False,
+              help='Disable the output coloring, useful for storing the output to file etc.')
+def stats_list_files(**kwargs):
+    """Show stat files stored in the stats directory (.perun/stats/). This command shows only a
+    limited number of the most recent files by default. This can be, however, changed by the
+    --top and --from-minor options.
+
+    The default output format is 'file size | minor version | file name'.
+    """
+    commands.list_stat_objects('files', **kwargs)
+
+
+@stats_group.command('list-versions')
+@click.option('--top', '-N', type=int, default=_DEFAULT_STATS_LIST_TOP, show_default=True,
+              help='Show only top N minor versions. Show all versions if set to 0. '
+                   'The minor version to start at can be changed using --from-minor.')
+@click.option('--from-minor', '-m', default=None, metavar='<hash>', is_eager=True,
+              callback=cli_helpers.lookup_minor_version_callback,
+              help='Show minor versions starting from a certain minor version (default is HEAD).')
+@click.option('--no-dir-size', '-d', flag_value=True, default=False,
+              help='Do not show the size of the version directory.')
+@click.option('--no-file-count', '-f', flag_value=True, default=False,
+              help='Do not show the number of files in each version directory.')
+@click.option('--no-total-size', '-t', flag_value=True, default=False,
+              help='Do not show the total size of all the versions combined.')
+@click.option('--sort-by-size', '-s', flag_value=True, default=False,
+              help='Sort the versions by size instead of their VCS order.')
+@click.option('--no-color', '-c', flag_value=True, default=False,
+              help='Disable the output coloring.')
+def stats_list_versions(**kwargs):
+    """Show minor versions stored as directories in the stats directory (.perun/stats/).
+    This command shows only a limited number of the most recent versions by default. This can be,
+    however, changed by the --top and --from-minor options.
+
+    The default output format is 'directory size | minor version | file count'.
+    """
+    commands.list_stat_objects('versions', **kwargs)
+
+
+@stats_group.group('delete')
+def stats_delete_group():
+    """Allows the deletion of stat files, minor versions or the whole stats directory.
+    """
+    pass
```

Comment:
I don't think I follow here. This is a click group for the delete commands of files, minor version directories or the entire stats directory, e.g. 'perun utils stats delete file <path>' / 'perun utils stats delete version <version dir>' / 'perun utils stats delete .' and it's not meant to be a TODO, it just doesn't have any options, only subcommands. The main reason for this is that it seems more convenient to write the commands as mentioned, instead of e.g. 'perun utils stats delete-file <...>' or 'delete-version' etc.

Reply:
My bad, I didn't realize it was a group ;). I though it was empty API for future implementation.